zombienet_tests/misc/0001-paritydb.feature

```diff
@@ -0,0 +1,102 @@
+Description: Check that paritydb works without affecting finality lag and block production.
+Network: ./0001-paritydb.toml
+Creds: config
+
+# Check configuration enables ParityDB.
+validator-0: log line contains "💾 Database: ParityDb" 
+validator-1: log line contains "💾 Database: ParityDb"
+validator-2: log line contains "💾 Database: ParityDb"
+validator-3: log line contains "💾 Database: ParityDb"
+validator-4: log line contains "💾 Database: ParityDb"
+validator-5: log line contains "💾 Database: ParityDb"
+validator-6: log line contains "💾 Database: ParityDb"
+validator-7: log line contains "💾 Database: ParityDb"
+validator-8: log line contains "💾 Database: ParityDb"
+validator-9: log line contains "💾 Database: ParityDb"
+
+validator-0: is up
+validator-1: is up
+validator-2: is up
+validator-3: is up
+validator-4: is up
+validator-5: is up
+validator-6: is up
+validator-7: is up
+validator-8: is up
+validator-9: is up
+
+# Check authority status and peers.
+validator-0: reports node_roles is 4
+validator-1: reports node_roles is 4
+validator-2: reports node_roles is 4
+validator-3: reports node_roles is 4
+validator-4: reports node_roles is 4
+validator-5: reports node_roles is 4
+validator-6: reports node_roles is 4
+validator-7: reports node_roles is 4
+validator-8: reports node_roles is 4
+validator-9: reports node_roles is 4
+
+# Ensure parachains are registered.
+validator-0: parachain 2000 is registered
+validator-0: parachain 2001 is registered
+validator-0: parachain 2002 is registered
+validator-0: parachain 2003 is registered
+validator-0: parachain 2004 is registered
+validator-0: parachain 2005 is registered
+validator-0: parachain 2006 is registered
+validator-0: parachain 2007 is registered
+validator-0: parachain 2008 is registered
+validator-0: parachain 2009 is registered
+
+# Check if network is fully connected.
+validator-0: reports peers count is at least 19 within 20 seconds
+validator-1: reports peers count is at least 19 within 20 seconds
+validator-2: reports peers count is at least 19 within 20 seconds
+validator-3: reports peers count is at least 19 within 20 seconds
+validator-4: reports peers count is at least 19 within 20 seconds
+validator-5: reports peers count is at least 19 within 20 seconds
+validator-6: reports peers count is at least 19 within 20 seconds
+validator-7: reports peers count is at least 19 within 20 seconds
+validator-8: reports peers count is at least 19 within 20 seconds
+validator-9: reports peers count is at least 19 within 20 seconds
+
+# Wait for progress.
+sleep 120 seconds
+
+# Ensure parachains made progress.
+validator-0: parachain 2000 block height is at least 10 within 30 seconds
```

Comment:
I'd prefer to remove sleep and check for height of 1-2 with a good enough timeout. We don't need to artificially make the tests take longer than they should. Esp. with async backing, this will change.